Directory.Packages.props

```diff
@@ -5,46 +5,46 @@
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>

```

Comment:
dotnet-version will need to be bumped in the yml files that define the build actions too,

Reply:
Added dual SDK installations everywhere to avoid dependency on default agent config.